src/platforms/platforms.ts

```diff
@@ -65,18 +109,55 @@ const platforms: Record<PlatformName, PlatformData> = {
     gatedEntity: "repo",
     cardPropsHook: useGithubCardProps,
     cardMenuComponent: GithubCardMenu,
+
+    oauth: {
+      url: "https://github.com/login/oauth/authorize",
+      params: {
+        client_id: process.env.NEXT_PUBLIC_GITHUB_CLIENT_ID,
+        scope: {
+          creation: "repo,read:user",
```

Comment:
I believe we actually need this scope for some of the requirements too, so not sure what could we do regarding this. If we'd request this scope from everyone, then the users wouldn't be too happy, so maybe we should send the "creation" scope in the reconnect button? (because we only show a reconnect button for the users who haven't authenticated with the required scope via GitHub - that's why we added that ternary a couple of months ago in `ConnectRequirementPlatformButton`)

Reply:
Thanks for pointing this out! This is an interesting scenario
I think I might rename the scope levels to _basic_ and _extended_ or something similar
I'm not sure whether we should first request _basic_ level and then request _extended_ when needed (on reconnect, as you recommended), or request for all the scopes, we will need 🤔 

It looks to me, that currently on the account modal, we request _basic_ scopes, but on a requirement card we request _extended_ scopes (for both reconnect and connect)

Reply:
For now I changed it to request _creation_ scopes on reauth in https://github.com/agoraxyz/guild.xyz/pull/858/commits/506ce2bdc9627878c31096607d02bdd756030770

Reply:
Great, I think it's good this way for now!